### PR TITLE
Removed dead code, added Makefile for MACOS and comments

### PR DESCRIPTION
**Pull request by SanyaMittal17**

### CODE REVIEW

File: bootmain.c
Comment (andTEJAsan) [inline bootmain.c:1]: In line 28, we are reading the ELF header of the kernel into memory, not the boot loader. The programme that is running is the boot loader. It is already loaded in memory

**Review by andTEJAsan — COMMENTED** [review]
In line 28 of bootmain.c, we are reading the ELF header of the kernel into memory, not the boot loader. The programme that is running is the boot loader. It is already loaded in memory

**Comment by heisenberg600** [comment]
Hey, you can not write comments explaining the code. If you want to explain something in the code, add it in the .md files only.

**Comment by SanyaMittal17** [comment]
> Hey, you can not write comments explaining the code. If you want to explain something in the code, add it in the .md files only.

**Comment by SanyaMittal17** [comment]
> Hey, you can not write comments explaining the code. If you want to explain something in the code, add it in the .md files only.

I'll update the .md file.

**Comment by heisenberg600** [comment]
Closing this PR as it contains one file addition "MAKEFILE(MCOS)" which is not allowed. Also contains the comments in the source code file

**Pull request closed without merging**